src/NUnitFramework/framework/Constraints/EachItemConstraintResult.cs

```diff
@@ -0,0 +1,40 @@
+namespace NUnit.Framework.Constraints
+{
+    /// <summary>
+    /// Provides a <see cref="ConstraintResult"/> for the constraints 
+    /// that are applied to each item in the collection
+    /// </summary>
+    public class EachItemConstraintResult : ConstraintResult
+    {
+        private readonly object _nonMatchingItem;
+        private readonly int _nonMatchingItemIndex;
+
+        /// <summary>
+        /// Constructs a <see cref="EachItemConstraintResult" /> for a particular <see cref="Constraint" />.
+        /// Only used for Failure
```

Comment:
I wouldn't normally be too worried, but given I've suggested other changes anyway, would you mind just tidying up the full-stops in the doc-comments? There's just a mix of some ending with full stops and some not at the moment. 😄

Reply:
I wanted to look in other framework classes, which approach is commonly used - with or without full-stops.
Seems like I unconsciously chose correct approach - just mix them whatever you like :))))

Reply:
Haha - yes, room for improvement in the codebase there too! 😉